.github/actions/deploy/deploy.py

```
@@ -0,0 +1,866 @@
+        print(f'🏷️  Deployment namespace: {self.deployment_namespace}')
+
+        # Create deployment namespace early since it's needed for secrets
+        print(f'🏷️  Creating deployment namespace: {self.deployment_namespace}')
+        self.run_command(
+            ['kubectl', 'create', 'namespace', self.deployment_namespace],
+            check=False  # Don't fail if namespace already exists
+        )
+
+    def run_command(self,
+                    cmd: List[str],
+                    cwd: str = None,
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)